src/Facebook/InstantArticles/Elements/LineBreak.php

```diff
@@ -44,8 +44,22 @@ public function toDOMElement($document = null)
         if (!$document) {
             $document = new \DOMDocument();
         }
+
+        if (!$this->isValid()) {
+            return $this->emptyElement($document);
+        }
+
         $br = $document->createElement('br');
 
         return $br;
     }
+
+    /**
+     * Overrides the @see TextContainer::isValid() to a always valid one, since
+     * <br> tag will never be "invalid".
+     */
+    public function isValid()
+    {
+        return true;
```

Comment:
Consider returning invalid if the BR contains any textChildren (as it'll not be rendered).